src/connector/src/parser/unified/avro.rs

```diff
@@ -269,6 +270,13 @@ impl<'a> AvroParseOptions<'a> {
             (Some(DataType::Jsonb), Value::String(s)) => {
                 JsonbVal::from_str(s).map_err(|_| create_error())?.into()
             }
+            (Some(DataType::Jsonb), v) => {
```

Comment:
Can we restrict the top level to be `Value::Map`?

Reply:
Oops it's a mistake. I had wanted to make it Map